pkg/telemetry/exporter/cli/cli.go

```diff
@@ -26,7 +26,7 @@ import (
 	"path"
 	"time"
 
-	uuid "github.com/satori/go.uuid"
+	"github.com/google/uuid"
```

Comment:
It's interesting that we use both. Can you make a tech debt issue to unify these ?